lib/robot.js

```diff
@@ -139,8 +139,9 @@ class Robot {
   async auth (id, log = this.log) {
     const github = new GitHubApi({
       debug: process.env.LOG_LEVEL === 'trace',
-      host: process.env.GHE_HOST || 'api.github.com',
-      pathPrefix: process.env.GHE_HOST ? '/api/v3' : '',
+      baseUrl: process.env.GHE_HOST ?
+        `https://${process.env.GHE_HOST}/api/v3` :
+        'https://api.github.com',
```

Comment:
You should be able to omit the second consequent (buzzword) of this ternary:

```
baseUrl: process.env.GHE_HOST && `https://${process.env.GHE_HOST}/api/v3`
```

or if that feels wrong:

```
baseUrl: process.env.GHE_HOST ? `https://${process.env.GHE_HOST}/api/v3` : null
```